### PR TITLE
Added Paradise procedures to docs v1.5 -- CASMTRIAGE-7234 / CASMHMS-6255

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
# Description

Additional documentation for Foxconn (Paradise) nodes
Script for moving FAS Foxconn firmware images from S3 to tftp server
Added Foxconn to spelling dictionary and updated docs using Foxconn

# Relates to:

* CASMTRIAGE-7234
* CASMHMS-6255

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Add_a_Standard_Rack_Node.md [hunk 1]
```diff
@@ -357,6 +357,8 @@ Follow the [Added Hardware](../network/management_network/added_hardware.md) pro
     ping "${BMC_XNAME}"
     ```
 
+1. If adding `Foxconn` (Paradise) nodes to the system, follow the [Replacing `Foxconn` Usernames and Passwords in Vault](./Replacing_Foxconn_User_Pass.md) procedure if nodes fail to discover.
```
Comment (jwlv) [inline operations/node_management/Add_a_Standard_Rack_Node.md]: Same here, can remove ` around Foxconn if you add Foxconn to .spelling in the root directory.

**Review by shunr-hpe — APPROVED** [review]

File: operations/firmware/Update_Firmware_with_FAS.md [hunk 2]
```diff
@@ -59,7 +59,8 @@ Table 1. Upgradable Firmware Items
 | Cray             | `chassisBMC` | `BMC`, `Recovery`                                                                                            |
 | Cray             | `routerBMC`  | `BMC`, `Recovery`                                                                                            |
 | Gigabyte         | `nodeBMC`    | `BMC`, `BIOS`                                                                                                |
-| HPE              | `nodeBMC`    | `iLO 5`, `iLO 6`, `System ROM` ,`Redundant System ROM`                                                                |
+| HPE              | `nodeBMC`    | `iLO 5`, `iLO 6`, `System ROM` ,`Redundant System ROM`                                                       |
+| `Foxconn`        | `nodeBMC`    | `bmc_active`, `bios_active`, `erot_active` ,`fpga_active`, `pld_active`                                      |
```
Comment (jwlv) [inline operations/firmware/Update_Firmware_with_FAS.md]: Adding ` around Foxconn causes it to render differently than the other manufacturers in the table.  No need for the ` if you add Foxconn to .spelling in the root directory.

File: operations/firmware/Update_Firmware_with_FAS.md [hunk 3]
```diff
@@ -123,6 +124,7 @@ After identifying which hardware is in the system, start with the top item on th
    1. [`iLO 5` (BMC)](FAS_Use_Cases.md#manufacturer-hpe--device-type-nodebmc--target-ilo-5-bmc)
    1. [`iLO 6` (BMC)](FAS_Use_Cases.md#manufacturer-hpe--device-type-nodebmc--target-ilo-6-bmc)
    1. [System ROM (BIOS)](FAS_Use_Cases.md#manufacturer-hpe--device-type-nodebmc--target-system-rom-bios)
+1. [`Foxconn` (Paradise)](FAS_Paradise.md)
```
Comment (jwlv) [inline operations/firmware/Update_Firmware_with_FAS.md]: Can remove ` around Foxconn if you add Foxconn to .spelling in the root directory.

**Review by jwlv — APPROVED** [review]
Just a few issues that could be resolved by adding Foxconn to .spelling

File: operations/node_management/Add_a_Standard_Rack_Node.md (hunk 1 above)
Comment (mbuchmann-hpe) [reply operations/node_management/Add_a_Standard_Rack_Node.md]: Added to dictionary

**Pull request merged**